decayAmplitude/isobarDecayTopology.cc

```diff
@@ -567,25 +567,32 @@ isobarDecayTopology::getIsospinSymmetrization()
 		}
 		// Some first checks if the permutation makes sense.
 		bool breaking = false;
-		for(unsigned int j = 0; j < map.size(); ++j) {
```

Comment:
In the original code, all permutations are checked against the identity, while in the new code, all but the first permutation are checked against the identity (and all previously accepted permutations).
If the first permutation is the identity, which I think it is, this does not matter. However, if the first permutation is a permutation that breaks w.r.t. the identity, this one would be accepted, while the identity would be dropped.